### DISCUSSION TITLE
Does v3.1.2 support redis6 ?

### DISCUSSION THREAD

**Discussion by marss112** [NodeRedis/node_redis#2379] [Q&A] [answered]
Hi all
Recently, the company has a project that uses the v3.1.2 redis package, but the redis version in the cloud is forced to be upgraded to redis6. We have done some tests, and the test is no problem so far, but I am still a little worried.
 Is there any risk if v3.1.2 is connected to redis6?

**Comment by leibale** [comment, marked as answer]
It should work just fine :)

edit: BTW, Redis 7 should work as well

**Reply by marss112** [reply]
thanks for the reply
